### PR TITLE
Add debug flag for printing library logic index of winning solution

### PR DESCRIPTION
**Pull request by benjaminulmer**
Mostly for development purposes.

### CODE REVIEW

File: Tensile/Source/lib/include/Tensile/MasterSolutionLibrary.hpp [hunk 1]
```diff
@@ -134,15 +135,21 @@ namespace Tensile
...
                 }
             }
             else
-                return library->findBestSolution(problem, hardware, fitness);
-        }
+                rv = library->findBestSolution(problem, hardware, fitness);
 
+            if(Debug::Instance().printLibraryLogicIndex() && rv)
+            {
+                std::cout << "Library logic solution index of winning solution: "
```
Comment (nielenventer) [inline Tensile/Source/lib/include/Tensile/MasterSolutionLibrary.hpp:148]: nit: I'd vote to have a separate message for no solution too, more info is probably better if we're debugging

Comment (benjaminulmer) [reply Tensile/Source/lib/include/Tensile/MasterSolutionLibrary.hpp:148]: Sure. The user would probably find out pretty quick anyways if no solution is found, but having a message for logging purposes could be nice. How's a simple "No solution found" sound?

**Review by nielenventer — APPROVED** [review]

**Review by babakpst — APPROVED** [review]
I built rocblas with this branch. It drops the correct solution index from the lib logic.

**Comment by benjaminulmer** [comment]
Precheckin gfx908 fails are also showing up in develop, so they're not related to this change.

**Pull request merged**